### DISCUSSION TITLE
Error when installing K2forJ4

### DISCUSSION THREAD

**Discussion by dailydealsuk** [eworkers/K2ForJ4#76] [General]
Hi, when I tried to install the K2forJ4 zip, I get the below error (from debug)

Call to undefined method Joomla\CMS\Factory::getContainer()
Call stack
--
# | Function | Location
1 | () | JROOT/components/com_k2/router.php:29
2 | K2Router->__construct() | JROOT/libraries/src/Router/SiteRouter.php:789
3 | Joomla\CMS\Router\SiteRouter->getComponentRouter() | JROOT/libraries/src/Router/SiteRouter.php:131
4 | Joomla\CMS\Router\SiteRouter->parse() | JROOT/libraries/src/Application/CMSApplication.php:1142
5 | Joomla\CMS\Application\CMSApplication->route() | JROOT/libraries/src/Application/SiteApplication.php:796
6 | Joomla\CMS\Application\SiteApplication->route() | JROOT/libraries/src/Application/SiteApplication.php:218
7 | Joomla\CMS\Application\SiteApplication->doExecute() | JROOT/libraries/src/Application/CMSApplication.php:225
8 | Joomla\CMS\Application\CMSApplication->execute() | JROOT/index.php:49

**Comment by eworkers** [comment]
Please update and check if the last commit (a8c07fd) fixes your issue, and please open issues in issue section next time and not in discussion board.

**Comment by dailydealsuk** [comment]
It does, it's working perfect, thankyou. 

| 
|  |  | 
| 
Claire Roach

Founder
 |
| 
  02920708301
  moneysavingcentral.co.uk
***@***.***
 |

 |

 |


| 
      
 |  |


 

    On Sunday, 21 April 2024 at 23:11:02 BST, eworkers ***@***.***> wrote:  
 
 


Please update and check if the last commit (a8c07fd) fixes your issue, and please open issues in issue section next time and not in discussion board.
